lib/oxrdfxml/src/error.rs

```diff
@@ -72,61 +39,39 @@ impl From<quick_xml::Error> for ParseError {
 }
 
 /// An error in the syntax of the parsed file.
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
+#[error(transparent)]
 pub struct SyntaxError {
+    #[from]
     pub(crate) inner: SyntaxErrorKind,
```

Comment:
Genuine question: Why not `SyntaxError(#[from] SyntaxErrorKind)` like in the other places?

Reply:
thx, missed it, fixed